File: VB/PivotGridCustomization.Module.Win/WinModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.ComponentModel

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Editors

Namespace PivotGridCustomization.Module.Win
	<ToolboxItemFilter("Xaf.Platform.Win")> _
	Public NotInheritable Partial Class PivotGridCustomizationWindowsFormsModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
	Protected Overrides Sub RegisterEditorDescriptors(ByVal editorDescriptors As List(Of DevExpress.ExpressApp.Editors.EditorDescriptor))
			MyBase.RegisterEditorDescriptors(editorDescriptors)
			editorDescriptors.Add(New DetailViewItemDescriptor(New DetailViewItemRegistration(GetType(IModelPivotGridCustomizationItem), GetType(PivotGridCustomizationDetailViewItem), True)))
	End Sub

	End Class
End Namespace

File: CS/PivotGridCustomization.Module.Win/WinModule.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Editors;

namespace PivotGridCustomization.Module.Win {
    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class PivotGridCustomizationWindowsFormsModule : ModuleBase {
        public PivotGridCustomizationWindowsFormsModule() {
            InitializeComponent();
        }
    protected override void RegisterEditorDescriptors(List<DevExpress.ExpressApp.Editors.EditorDescriptor> editorDescriptors) {
            base.RegisterEditorDescriptors(editorDescriptors);
            editorDescriptors.Add(new ViewItemDescriptor(new ViewItemRegistration(typeof(IModelPivotGridCustomizationItem), typeof(PivotGridCustomizationDetailViewItem), true)));
    }

    }
}
